testing/trino-testing/src/main/java/io/trino/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -39,7 +39,7 @@
  * @see AbstractTestDistributedQueries
  */
 public abstract class AbstractTestIntegrationSmokeTest
```

Comment:
Why not to go other way round and slowly migrate tests to `AbstractTestDistributedQueries` and at some point we could simply remove `AbstractTestIntegrationSmokeTest `.

`AbstractTestDistributedQueries` allows developer to select what test they want to run. Maybe we could have artificial feature like `SMOKE`, so one could run some some `SMOKE` tests if they want to test something that affects entire connector.

Reply:
> AbstractTestDistributedQueries allows developer to select what test they want to run.

how do i do this?

Reply:
You can extend this class and override methods like `supportsDelete()`

Reply:
Oh. i understand now. And?

This change is an incremental improvement to the current state. It improves testing for read-only connectors, while also removes some duplicated tests for read/write connectors (testShowSchemas).